admin/src/scenes/phase0/components/CniField.jsx

```diff
@@ -38,6 +40,7 @@ export function CniField({
   const [requestButtonClass, setRequestButtonClass] = useState("");
   const [mouseIn, setMouseIn] = useState(false);
   const [cniModalOpened, setCniModalOpened] = useState(false);
+  const datePassed = young.latestCNIFileExpirationDate ? new Date(young.latestCNIFileExpirationDate) < new Date() : false;
```

Comment:
Par convention, les booléens commencent par is ou has, que penses tu de la nommer isDatePassed ? En suivant ces conventions on comprend tout de suite le type de la variable.

Reply:
ok plus clair en effet